### ISSUE TITLE
Mat-autocomplete overlay height is not correct

### ISSUE THREAD

**Issue by jainankit102** [angular/material2#11762]
#### Bug, feature request, or proposal:
Bug
Mat-autocomplete overlay height is not correct proper when user working on 200% browser resolution.

#### What is the expected behavior?
It should come under selected resolution screen. Overlay height should be adjusted as screen resolution.

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>
Everywhere, check official example at 200% resolution with minimum 10 option.

#### What is the use-case or motivation for changing an existing behavior?
Accessibility are facing hard to identify

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
5.2 and 6.0+

#### Is there anything else we should know?
![image](https://user-images.githubusercontent.com/24789588/41291740-27c5f42a-6e6e-11e8-8e45-477aff67835f.png)

My Analysis: By default for every screen resolution mat-autocomplete-panel has max-height 256px so if user in higher resolution above 200% then it is getting out from the screen. Overlay height should be adjusted as screen resolution.

**Comment by crisbeto** [comment]
By the looks of it, this is behaving as expected. The logic for placing the panel is that it puts it above or below, depending on which position will show the largest area of the panel. We also have an API for pushing the panel into the viewport if it can't fit, but it's currently disabled because of https://github.com/angular/material2/pull/11628.
